### PR TITLE
Extract code examples from the `/dao` page

### PR DESCRIPTION
**Pull request by onetom**
so they
* can be edited with syntax highlight
* can be linted
* can even be tested automatically eventually

There is a missing line in the `Congress` contract which I wanted to fix, but I thought such fixes can be made easier in the future if the source code lives in their own separate files.

### CODE REVIEW

File: views/content/dao.md [hunk 1]
```diff
@@ -18,316 +18,9 @@ The way this particular democracy works is that it has an **Owner** which works
...
-
-            // Fire Events
-            ProposalTallied(proposalNumber, p.currentResult, p.numberOfVotes, p.proposalPassed);
-        }
-    }
-
-
-
-
+```solidity
```
Comment (alexvandesande) [inline views/content/dao.md]: This doesn't color code solidity for me. Leaving it as **\`\`\`**  or even **\`\`\`js** does the trick. Have you added a color code plugin for solidity?

**Comment by ryanio** [comment]
I like this idea of extracting all the solidity code into .sol files

**Comment by alexvandesande** [comment]
Thanks for that! I wanted to do this for a while but couldn't figure out how

**Comment by ryanio** [comment]
Sweet!

File: views/content/dao.md (hunk 1 above)
Comment (onetom) [reply views/content/dao.md]: No, I haven't added any plugin for it, but I thought putting `solidity` there doesn't do any harm, BUT might inspire someone to make it happen :)
I didn't want to mark it `js` when it's not and that highlighting was a bit wonky anyway.

**Pull request merged**